cmd/crates/soroban-rpc-client/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-name = "soroban-rpc"
+name = "soroban-rpc-client"
```

Comment:
I think we should update the `soroban-rpc` crate's readme on https://crates.io/crates/soroban-rpc somehow to point to the new crate name. I think that the only way to do that is to publish a new version of it though. Maybe we make a soroban-rpc crate in this repo that is empty other than a readme and publish it?

Reply:
Yes thats the only option seems like. publish empty soroban-rpc with updated readme and gonna yank it just to deprecate.